README.md

```diff
@@ -153,18 +149,18 @@ or a callable for each record:
 ```php
 public function export()
 {
-    $posts = $this->Post->find('all');
+    $posts = $this->Posts->find();
     $_serialize = 'posts';
     $_header = ['Post ID', 'Title', 'Created'];
     $_extract = [
         'id',
         function ($row) {
```

Comment:
Should we annotate the entity here? `(Post $row)` makes title attribute auto-completably.
I always do this in my code :)